### DISCUSSION TITLE
Multiple lets encrypt domains?

### DISCUSSION THREAD

**Discussion by jdixon-86** [requarks/wiki#6097] [Help / Questions] [answered]
I am using the LETSENCRYPT_DOMAIN argument and I know there are some options such as SUBDOMAINS= and EXTRA_DOMAINS= but I was not able to get these to work. Are there any supported methods to deploying via docker on digitalocean with multiple domains in the certificate?

**Comment by NGPixel** [comment, marked as answer]
No. Use a reverse proxy like nginx if you need a more complex setup.
